client/src/controllers/DismissibleController.ts

```diff
@@ -0,0 +1,51 @@
+import { Controller } from '@hotwired/stimulus';
+import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
+
+/**
+ * Adds the ability to make an element dismissible so that it updates it's class and makes an async request.
+ *
+ * @example
+ * <section
+ *  aria-labelledby="whats-new-heading"
+ *  data-controller="w-dismissible"
+ *  data-w-dismissible-dismissed-class="w-dismissible--dismissed"
+ *  data-w-dismissible-id-value="Whats new in Wagtail"
+ * >
+ *  <button type="button" data-action="w-dismiss#dismissible">Close</button>
+ * </section>
+ */
```

Comment:
This JSDoc above, it should be on the controller class, maybe add a different one.foe.the update function.